src/material-experimental/mdc-helpers/_mdc-helpers.scss

```diff
@@ -0,0 +1,205 @@
+@import '@material/feature-targeting/functions';
```

Comment:
Can we split this fill up based on types of thing? I want to avoid any of of generic "helpers" or "utils" type artifacts. It looks like a big chunk of it could go under "typography" and the rest under "theming"

Reply:
It's actually kind of difficult to split this up right now because of our release process (unless I want each file to be its own secondary entry point). I added a `TODO` to change it when we switch to bazel